core/src/organization/organization/organization.controller.spec.ts

```diff
@@ -0,0 +1,20 @@
+import type {TestingModule} from "@nestjs/testing";
```

Comment:
Delete this? We don't have test files for any other controller, service, or resolver.